### DISCUSSION TITLE
Can we train the model with our own data?

### DISCUSSION THREAD

**Discussion by Bert912** [currentslab/extractnet#10] [Q&A]
I would like to expand on the model by training it on more data, but it doesn't look like I can do that as it is.

**Comment by theblackcat102** [comment]
@Bert912  Hi, the underlying model is based on [dragnet](https://github.com/dragnet-org/dragnet#training-content-extraction-models) so you can easily implement your own training data and swap in to use it. The only change of extractnet from dragnet was using neural networks instead of a decision tree.

**Reply by BradKML** [reply]
Wait does that mean the performance will be slowed down?
